schema/schema2025-08-22.yaml

```
@@ -0,0 +1,1152 @@
+                    content: Yo momma so stupid, she went to the dentist to get Bluetooth.
+                    status: Pending
+                    reports_count: 2
+                  summary: List of User's Insults
+                  description: Shows a simplified list of insults created by the user
+          description: ''
+  /api/insult/{reference_id}:
+    get:
+      operationId: retrieve_insult
+      description: Retrieve a specific insult by ID.
+      parameters:
+      - in: path
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)